PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-applicationinsights/src/main/java/com/microsoft/azure/toolkit/intellij/applicationinsights/creation/ApplicationInsightsCreationDialog.java

```diff
@@ -0,0 +1,107 @@
+import com.microsoft.azure.toolkit.lib.applicationinsights.ApplicationInsightDraft;
+import com.microsoft.azure.toolkit.lib.applicationinsights.AzureApplicationInsights;
+import com.microsoft.azure.toolkit.lib.common.form.AzureForm;
+import com.microsoft.azure.toolkit.lib.common.form.AzureFormInput;
+import com.microsoft.azure.toolkit.lib.common.model.AzResource;
+import com.microsoft.azure.toolkit.lib.common.model.Region;
+import com.microsoft.azure.toolkit.lib.common.model.ResourceGroup;
+import com.microsoft.azure.toolkit.lib.common.model.Subscription;
+import org.apache.commons.lang.StringUtils;
+import org.jetbrains.annotations.Nullable;
+
+import javax.swing.*;
+import java.awt.event.ItemEvent;
+import java.util.Arrays;
+import java.util.List;
+
+public class ApplicationInsightsCreationDialog extends AzureDialog<ApplicationInsightDraft> implements AzureForm<ApplicationInsightDraft> {
+    private static final String DIALOG_TITLE = "Create Application Insight";
+
+    private JPanel pnlRoot;
+    private SubscriptionComboBox subscriptionComboBox;
+    private ResourceGroupComboBox resourceGroupComboBox;
+    private RegionComboBox regionComboBox;
+    private InsightNameTextField txtName;
+
+    public ApplicationInsightsCreationDialog(Project project) {
+        super(project);
+        init();
+        initListeners();
+    }
+
+    @Override
+    public AzureForm<ApplicationInsightDraft> getForm() {
+        return this;
+    }
+
+    @Override
+    protected String getDialogTitle() {
+        return DIALOG_TITLE;
+    }
+
+    @Override
+    protected @Nullable JComponent createCenterPanel() {
+        return pnlRoot;
+    }
+
+    @Override
+    public ApplicationInsightDraft getValue() {
+        final Subscription subscription = subscriptionComboBox.getValue();
+        final ResourceGroup resourceGroup = resourceGroupComboBox.getValue();
+        final String name = txtName.getValue();
+        final Region region = regionComboBox.getValue();
+        final ApplicationInsightDraft result =
+                Azure.az(AzureApplicationInsights.class).forSubscription(subscription.getId()).applicationInsights().create(name, resourceGroup.getName());
+        result.setRegion(region);
+        return result;
+    }
+
+    @Override
+    public void setValue(ApplicationInsightDraft data) {
+        final String resourceName = StringUtils.equals(data.getName(), AzResource.NONE.getName()) ? StringUtils.EMPTY : data.getName();
+        this.txtName.setValue(resourceName);
+        this.subscriptionComboBox.setValue(data.getSubscription());
+        // todo: @hanli support draft resource group
+        this.resourceGroupComboBox.setValue(data.getResourceGroup().toPojo());
+        this.regionComboBox.setValue(data.getRegion());
+    }
+
+    @Override
+    public List<AzureFormInput<?>> getInputs() {
+        return Arrays.asList(subscriptionComboBox, resourceGroupComboBox, txtName, regionComboBox);
```

Comment:
make `txtName` the first element.
so that it will be automatically focused when the dialog is opened.